tables/hbo.tbl

```diff
@@ -0,0 +1,208 @@
+#https://huc.edu/library_blog/learning-about-hebrew-braille/#:~:text=The%20Code%20created%20by%20Brevis,books%20written%20in%20Hebrew%20Braille.
+
+#-maintainer: Paul Geoghegan <contact@envisionly.tech>
+#-maintainer: Matityhau Yeshurun <yeatersink@gmail.com>
+
+# always op code characters
+always ִי 35  # hiriq-yod
+always ֵי 3456  # tsere-yod
+always ּו 136  # shuruq
+always ּוְ 136-3  # shuruq sheva
+always בְּ 12-3  # bet with dagesh sheva
+always בּ 12  # bet with dagesh
+always דּ 5-145  # dalet with dagesh
+always דְּ 5-145-3  # dalet with dagesh sheva
+always וֹ 246  # holam-vav
+always וּ 136  # shuruq
+always יּ 5-245  # Yod with dagesh
+always יְּ 5-245-3  # Yod with dagesh sheva
+always כּ 13  # kaf with dagesh
+always כְּ 13-3  # kaf with dagesh sheva
+always לּ 5-123  # lamed with dagesh
+always לְּ 5-123-3  # lamed with dagesh sheva
+always מּ 5-134  # mem with dagesh
+always מְּ 5-134-3  # mem with dagesh sheva
+always נּ 5-1345  # nun with dagesh
+always נְּ 5-1345-3  # nun with  dagesh sheva
+always פּ 5-1234  # pe with dagesh
+always פְּ 5-1234-3  # pe with dagesh sheva
+always קּ 5-12345  # qof with dagesh
+always קְּ 5-12345-3  # qof with dagesh sheva
+always שְׁ 146-3  #  Shin with dot and sheva
+always שְׂ 156-3  #  Sin sheva
+always שׁ 146  # shin with dot
+always שׂ 156  # sin
+always ת 1456  # tav
+always תּ 2345  # tav with dagesh
+always תְּ 2345-3  # tav with dagesh sheva
+always בּ 12  # Bet with dagesh
+always בְּ 12-3  # Bet with dagesh sheva
+always כּ 13  # KAF with dagesh 
+always כְּ 13-3  # KAF with dagesh  sheva
+always תּ 5-1456  # tav with daggesh
+always תְּ 5-1456-3  # tav with daggesh sheva
+
+# letter op code characters
+letter ֒ 46-234  # segol - isolated
+letter ֓ 46-146  # shalshelet
+letter ֔ 2  # zaqef qatan
+letter ֕ 46-1245  # zaqef gadol
+letter ֖ 46-2345  # tipeha
+letter ֗ 46-1235  # revia
+letter ֘ 46-145  # zarqa
+letter ֙ 46-34  # pashta
+letter ֚ 46-245  # yetiv
+letter ֛ 46-1236  # tevir
+letter ֜ 46-1  # geresh
+letter ֝ 46-16  # geresh muqdam
+letter ֞ 46-12  # gershayim
+letter ֟ 46-1345  # qarney para
+letter ֠ 46-2356  # telisha gedola
+letter ֡ 46-236  # pazer
+letter ֢ 46-23  # atnah hafukh
+letter ֣ 46-136  # munah
+letter ֤ 46-125  # mahapakh
+letter ֥ 46-134  # merkha
+letter ֦ 46-13  # merkha kefula
+letter ֧ 46-145  # darga
+letter ֨ 46-1234  # qadma
+letter ֩ 46-12345  # telisha qetana
+letter ֪ 46-13456  # yerah ben yomo
+letter ֫ 46-135  # ole
+letter ֬ 46-24  # iluy
+letter ֭ 46-24  # dehi
+letter ֮ 46-1356  # zinor
+letter ֯ 456-134  # masora circle
+letter ְ 3  # sheva
+letter ֱ 26  # hataf segol
+letter ֲ 25  # hataf patah
+letter ֳ 345  # hataf qamats
+letter ִ 24  # hiriq
+letter ֵ 34  # tsere
+letter ֶ 15  # segol
+letter ַ 14  # patah
+letter ָ 126  # qamats
+letter ֹ 456-246  # holam Malay
+letter ֺ 135  # holam haser
+letter ֻ 346  # qubuts
+letter ּ 5  # dagesh
+letter ֽ 4  # meteg
+letter ֿ 45  # rafe
+letter ׁ 456-4  # shin dot
+letter ׂ 456-45  # sin dot
```

Comment:
I have the sense that brevity is not your main concern with this table, but I think that a two-character definition for the sin dot and shin dot on their own seems like it would be confusing. sin and shin already have their own braille characters, and although the sin dot and shin dot have their own Unicode characters, they don't provide any extra information except for indicating if the letter they are attached to is a sin or a shin. If you get to the letter sin in a text and you read the braille dots indicating that it's a sin, what purpose does this extra 456-45 have? 

I learned from reading the other Hebrew braille table that there are different ways to represent this letter in Unicode (e.g. "shin" and "shin with separate shin dot") but it seems to me that, when there are multiple ways to represent the same letter in Unicode, all of them should map to the same character in braille, just like they map to the same grapheme in print.